src/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8
+WORKDIR /app
+
+COPY . .
+
+ENV PYTHONUNBUFFERED=1
+
+RUN pip install --upgrade pip && pip install -r requirements.txt
+
+EXPOSE 8080
+
+ENTRYPOINT [ "sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8080" ]
```

Comment:
Add EoF(End-of-File), need to add one empty line at the end 
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_3_write_a_manifest/assets/87720583/67c4f69e-4298-4929-9423-535e96534822)